internal/campaigns/bind_workspace_test.go

```diff
@@ -54,7 +55,7 @@ func TestWorkspaceCreator_Create(t *testing.T) {
 
 		testTempDir := workspaceTmpDir(t)
 
-		creator := &WorkspaceCreator{dir: testTempDir, client: client}
+		creator := &dockerBindWorkspaceCreator{dir: testTempDir, client: client}
```

Comment:
I can't comment on it, but I think the function should be renamed from `func TestWorkspaceCreator_Create` to `func TestDockerBindWorkspaceCreator_Create`

Reply:
Yep, agreed.